js/index.js

```diff
@@ -8,35 +8,126 @@ const siteContent = {
+    "services-h4": "Services",
+    "services-content":
+      "Services content elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi. Suspendisse eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.",
+    "product-h4": "Product",
+    "product-content":
+      "Product content elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi. Suspendisse eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis.",
+    "vision-h4": "Vision",
+    "vision-content":
+      "Vision content elementum magna eros, ac posuere elvit tempus et. Suspendisse vel tempus odio, in interdutm nisi. Suspendisse eu ornare nisl. Nullam convallis augue justo, at imperdiet metus scelerisque quis."
   },
-  "contact": {
-    "contact-h4" : "Contact",
-    "address" : "123 Way 456 Street Somewhere, USA",
-    "phone" : "1 (888) 888-8888",
-    "email" : "sales@greatidea.io",
-  },
-  "footer": {
-    "copyright" : "Copyright Great Idea! 2018"
+  contact: {
+    "contact-h4": "Contact",
+    address: "123 Way 456 Street Somewhere, USA",
+    phone: "1 (888) 888-8888",
+    email: "sales@greatidea.io"
   },
+  footer: {
+    copyright: "Copyright Great Idea! 2018"
+  }
 };
 
 // Example: Update the img src for the logo
 let logo = document.getElementById("logo-img");
-logo.setAttribute('src', siteContent["nav"]["img-src"])
+logo.setAttribute("src", siteContent["nav"]["img-src"]);
+
+//Updates the nav element
+const nav = Array.from(document.querySelector("nav").children),
+  navTexts = [
+    siteContent["nav"]["nav-item-1"],
+    siteContent["nav"]["nav-item-2"],
+    siteContent["nav"]["nav-item-3"],
+    siteContent["nav"]["nav-item-4"],
+    siteContent["nav"]["nav-item-5"],
+    siteContent["nav"]["nav-item-6"]
+  ],
+  navElems = nav.map((elem, i) => (elem.textContent = navTexts[i]));
+
+const navContainer = document.querySelector("nav");
+const newNavOne = document.createElement("a");
+const newNavTwo = document.createElement("a");
+
+newNavOne.setAttribute.href = "#";
+newNavOne.textContent = "Portfolio";
+navContainer.appendChild(newNavOne);
+
+newNavTwo.setAttribute.href = "#";
+newNavTwo.textContent = "Charity";
+navContainer.appendChild(newNavTwo);
+
+const a = Array.from(document.querySelectorAll("a"));
+a.map(elem => (elem.style.color = "green"));
+
+//Updates the CTA
+siteContent.cta.h1 = "DOM\nIs\nAwesome";
+const ctaHeader = document.querySelector(".cta .cta-text h1"),
+  ctaBtn = document.querySelector(".cta .cta-text button"),
+  ctaImg = document.querySelector(".cta img");
+
+ctaHeader.textContent = siteContent["cta"]["h1"];
+ctaHeader.style.whiteSpace = "pre";
+ctaBtn.textContent = siteContent["cta"]["button"];
+ctaImg.src = siteContent["cta"]["img-src"];
```

Comment:
If you find the `["string"]` tedious, you can also just use dot notation.  I personally tend to only use brackets so I can place a variable in them.